.docs/index.md

```diff
@@ -138,7 +141,7 @@ properly. For instance, Docker communicates to the REX-Ray Volume Driver via
 a UNIX socket file.
 
 ```bash
-$ rexray start
+$ rexray service start
```

Comment:
I too prefer the ```service``` command because of what you noted. Hidden commands shouldn't be the preferred choice in the docs